src/EnergyPlus/ThermalComfort.cc

```diff
@@ -1882,12 +1882,12 @@ namespace ThermalComfort {
 
         X += H2;
 
-        for (J = 1; J <= 2; ++J) {
+        for (J = 0; J < 2; ++J) {
```

Comment:
I changed the bounds on loops such as this where possible to eliminate the 1-based vs 0-based `- 1` offset.